### PR TITLE
Added ability to resume from a run that crashed or completed

### PR DESCRIPTION
**Pull request by carolius**
## Description
Added ability to resume from a run that crashed or completed

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
Please describe the tests that you ran to verify your changes. 

- [ ] I have run the local tests with `pytest --gpus=X`.
- [ ] I have run the experiment with `EI_test_up.yaml`

### CODE REVIEW

**Comment by micahcarroll** [comment]
Lmk if you verify that can be re-run with the same exact sbatch command

**Comment by carolius** [comment]
> Lmk if you verify that can be re-run with the same exact sbatch command

It doesn't work like that currently, I could probably make it work but if you've redone the configs maybe i should do it on the new setup rather than the old one?

**Comment by micahcarroll** [comment]
Changed this branch to build off of my config branch

File: influence_benchmark/RL/base_iteration.py [hunk 1]
```diff
@@ -81,9 +81,41 @@ def __init__(
...
+                    self.lora_path = self.get_checkpoint_path(self.start_iteration - 2)
+
+        else:
+            self.start_iteration = 0
+            self.resume = False
+            self.trajectory_dir.mkdir(parents=True, exist_ok=False)
 
     def _save_kwargs(self, kwargs):
         self.kwargs_to_save = {k: v for k, v in kwargs.items() if k != "self"}
+        print(self.trajectory_dir)
```
Comment (micahcarroll) [inline influence_benchmark/RL/base_iteration.py:118]: Probably remove

**Pull request merged**